proposals/libocispec.md

```diff
@@ -0,0 +1,37 @@
+# OCI libocispec project proposal
+
+## Abstract
+Provide a C library for parsing and accessing OCI Runtime and Image specification JSON.
+
+## Proposal
+With repositories under the https://github.com/opencontainers organization:
+
+Rename (transfer) https://github.com/giuseppe/libocispec to https://github.com/opencontainers/libocispec.
```

Comment:
What about moving it here:
https://github.com/opencontainers/runtime-tools   maybe in contrib/libocispec/c/

Reply:
> What about moving it here:
https://github.com/opencontainers/runtime-tools maybe in contrib/libocispec/c/

My concern with that is that versioning the repo becomes tricky.  Do you lock the Go and C APIs together under a single version?  Version them separately with prefixed tags for C releases?  And you'd need to check with runtime-tools maintainers to see if enough of them care about and feel comfortable enough with C to maintain this as well.  I don't have a problem with this becoming an OCI Project in its own right, or remaining outside the OCI umbrella.  But I think stuffing it into an existing OCI Project may be more trouble than its worth.

And however this shakes out, we probably want to link the library from [here][1] to raise awareness.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/implementations.md#testing--tools